lemur/plugins/lemur_linuxdst/remote_host.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+from lemur.certificates import service
+import os
+
+
+def createCert(name, tempFolder, exportType):
+
+    lemCert = service.get_by_name(name)
+    if not os.path.exists(tempFolder):
+        os.mkdir(tempFolder)
+    if not os.path.exists('{0}/{1}'.format(tempFolder, lemCert.cn)):
+        os.mkdir('{0}/{1}'.format(tempFolder, lemCert.cn))
+    certFile = '{0}/{1}/cert.pem'.format(tempFolder, lemCert.cn)
+    keyFile = '{0}/{1}/priv.key'.format(tempFolder, lemCert.cn)
```

Comment:
@kevgliss  have you had any luck using this library with a non-standard sshd port?  It keeps yelling at me and I cannot find out what known_hosts it is referencing. I can use ssh to connect to the target machine with the standard ssh command no problem. I tired /etc/ssh/known_hosts and ~/.ssh/knownhosts

paramiko.ssh_exception.SSHException: Server '[target.domain.com]:2299' not found in known_hosts